src/utp_stream.cpp

```diff
@@ -137,7 +137,7 @@ enum
 	ACK_MASK = 0xffff,
 
 	// the number of packets that'll fit in the reorder buffer
-	max_packets_reorder = 512,
+	max_packets_reorder = 4096,
```

Comment:
Do you know if this is used in any context where it needs to be a compile-time constant?
If not, perhaps you could remove this constant and replace it with ``std::max(3, m_in_buf_size / 1400)`` (or something like that)

Reply:
`max_packets_reorder` is only used once, and only in`utp_stream.cpp`.
I assume it is safe to do as you suggest and make it depend on `m_in_buf_size`.
The only potential downside is addition of tiny computational overhead, but that is neglible.
I will redo the patch and upload new version.